src/main/java/com/sparta/newspeed/domain/comment/Comment.java

```diff
@@ -17,13 +19,18 @@ public class Comment extends TimeStamped {
     private Long id;
 
     @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "boardId")
+    @JoinColumn(name = "board_id")
     private Board board;
 
     @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "userId")
+    @JoinColumn(name = "user_id")
     private User user;
 
     @Column(nullable = false)
     private String content;
+
+    public void saveComment(String content, Board board) {
+        this.content = content;
+        this.board = board;
+    }
```

Comment:
public void saveComment(String content, Board board, User jwtUser) {
   this.content = content;
   this.board = board;
   this.user = jwtUser;
}